packages/web-components/src/components/cbp-list/cbp-list.tsx

```diff
@@ -10,9 +10,15 @@ export class CbpList {
   
   private renderedTag: HTMLElement;
 
+  /** Specifies the variant of list (simple, link, special) */
+  @Prop() variant: 'simple' | 'link'; 
+  
   /** Specifies the semantic tag to be rendered. */
   @Prop() tag: 'ul' | 'ol' = 'ul';
 
+  /** Specifies the font size for the list */
+  @Prop() size: 'sm' | 'lg';
```

Comment:
I would make this "normal" (it's not actually small) and "large" , defaulting to normal. It should also be reflected if you are using it as a CSS hook.